jupyterhub/values.schema.yaml

```diff
@@ -2757,6 +2757,15 @@ properties:
           See [the Kubernetes
           documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls)
           for more details about annotations.
+      extraPaths:
+        type: array
+        description: |
+          A list of custom paths to be added to the ingress configuration. 
+          Each custom path allows traffic to be routed to a specific backend service.
+          For each path, you can specify the service name, and either a service port number or a service port name.
```

Comment:
This can probably be omitted if we render like `{{- . | toYaml | nindent 10 }}`, and then we can lean on the kubernetes docs entirely.